corehq/pillows/sms.py

```diff
@@ -58,18 +57,16 @@ def calc_meta(cls):
         return cls.calc_mapping_hash({"es_meta": cls.es_meta, "mapping": cls.default_mapping})
 
     def change_transport(self, doc_dict):
-        domain = doc_dict.get('domain')
-        if domain and SQL_SMS_PILLOW.enabled(domain):
-            return
-
-        super(SMSPillow, self).change_transport(doc_dict)
+        # SMS changes don't go to couch anymore. Let the SqlSMSPillow process
+        # changes from now on.
+        # Also, we explicitly need this to be a no-op because we're going to
+        # delete all sms from couch and don't want them to be deleted from
+        # elasticsearch.
+        return
```

Comment:
@czue I wasn't sure how much of the old pillow i still need to keep around and how much can go?